platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1805,6 +1856,65 @@
 			name = Accounts;
 			sourceTree = "<group>";
 		};
+		9C6E280822A97F6B0056B7BE /* Categories */ = {
+			isa = PBXGroup;
+			children = (
+				9C6E280B22A97FDB0056B7BE /* CLLocation+MMEMobileEvents.h */,
+				9C6E280E22A97FDB0056B7BE /* CLLocation+MMEMobileEvents.m */,
+				9C6E281022A97FDB0056B7BE /* CLLocationManager+MMEMobileEvents.h */,
+				9C6E281222A97FDB0056B7BE /* CLLocationManager+MMEMobileEvents.m */,
+				9C6E281122A97FDB0056B7BE /* MMECategoryLoader.h */,
+				9C6E280922A97FDB0056B7BE /* MMECategoryLoader.m */,
+				9C6E280C22A97FDB0056B7BE /* NSData+MMEGZIP.h */,
+				9C6E280D22A97FDB0056B7BE /* NSData+MMEGZIP.m */,
+				9C6E280A22A97FDB0056B7BE /* UIKit+MMEMobileEvents.h */,
+				9C6E280F22A97FDB0056B7BE /* UIKit+MMEMobileEvents.m */,
+			);
+			path = Categories;
+			sourceTree = "<group>";
+		};
+		9C6E284A22A984710056B7BE /* scripts */ = {
+			isa = PBXGroup;
+			children = (
+				9C6E284B22A984710056B7BE /* install-packaging-dependencies.sh */,
+				9C6E284C22A984710056B7BE /* ios-code-coverage.sh */,
+				9C6E284D22A984710056B7BE /* release-notes-github.md.ejs */,
+				9C6E284E22A984710056B7BE /* document.sh */,
+				9C6E284F22A984710056B7BE /* check-events-symbols.sh */,
+				9C6E285022A984710056B7BE /* deploy-packages.sh */,
+				9C6E285122A984710056B7BE /* publish.sh */,
+				9C6E285222A984710056B7BE /* validate-framework-zip.sh */,
+				9C6E285322A984710056B7BE /* metrics.sh */,
+				9C6E285422A984710056B7BE /* package.sh */,
+				9C6E285522A984710056B7BE /* trigger-external-deploy-steps.sh */,
+				9C6E285622A984710056B7BE /* release-notes-jazzy.md.ejs */,
+				9C6E285722A984710056B7BE /* deploy-snapshot.sh */,
+				9C6E285822A984710056B7BE /* release-notes.js */,
+			);
+			path = scripts;
+			sourceTree = "<group>";
+		};
+		9C6E285922A9849E0056B7BE /* scripts */ = {
```

Comment:
This evidently adds the “scripts” folder twice:

<img width="272" alt="Screen Shot 2019-06-06 at 10 54 42 AM" src="https://user-images.githubusercontent.com/1198851/59054982-aee8e700-8849-11e9-80cd-2d5f7ff8a708.png">

Reply:
Also, if you could please capitalize Scripts, that’d soothe my frazzled pedant nerves. 🙇

Reply:
It's lower-case on the filesystem

Reply:
But we can be case-insensitive in our usage